src/views/v2/components/loader.php

```diff
@@ -22,8 +22,6 @@ class="tribe-events-view-loader tribe-common-a11y-hidden"
 		<?php esc_html_e( 'Loading view.', 'the-events-calendar' ); ?>
 	</span>
 	<div class="tribe-events-view-loader__dots tribe-common-c-loader">
-		<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--first"></div>
-		<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--second"></div>
-		<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--third"></div>
+		<?php $this->template( 'components/icons/dots', [ 'classes' => [ 'tribe-common-c-loader__dot-svg' ] ] ); ?>
```

Comment:
same note as in common, we should have 3 dot templates rather than one dots template so we aren't locking the structure in.